src/components/grid-aware/StatsBlock/StatsBlock.stories.jsx

```diff
@@ -7,11 +7,11 @@ export default {
   component: StatsBlock,
 };
 
-const Template = ({ title, subtitle, statCards, theme, titleColor }) => (
+const Template = ({ title, subtitle, barStatCards, theme, titleColor }) => (
```

Comment:
You'll want to revert all these changes if you split out the bar starts into a separate component, but for future reference, if we ever did want to add a completely different variant to an existing component, we should just define a new "story" instead of changing the existing one. This way, all the stories for a given component can be viewed in Storybook.

Here's an example from the Storybook docs on defining multiple stories: https://storybook.js.org/docs/react/writing-stories/introduction#using-args